globus_automate_client/cli/auth.py

```diff
@@ -161,6 +162,26 @@ def default(x):
                 )
         self.modified = False
 
+    def clear_tokens(
+        self,
+        environment_aware: bool = True,
+        callback: Optional[Callable[[str, TokenSet], bool]] = None,
+    ) -> None:
+        if environment_aware:
+            tokens = self.tokens_for_environment
+        else:
+            tokens = self.tokens
+        for scope, token_set in copy.copy(tokens).items():
+            if scope.startswith("__"):
+                continue
```

Comment:
Is there a certain format for the token storage? Would you be able to write a docstring / just describe what the expected format is? Mainly for the key since.

Reply:
I added a bit of doc and a constant to hopefully clearup the use of the `"__"` string.